### PR TITLE
add a blog post about our live transcript demo from Astricon 19

### PR DESCRIPTION
**Pull request by pc-m**

### CODE REVIEW

**Comment by bloom1** [comment]
I would like a second pass from @sduthil if possible :smiley:

**Comment by sduthil** [comment]
Fixed some typos and added a link to the presentations slides. Thank you Pascal for the post!

File: content/blog/live-transcript-demo-astricon-19.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Title: Live Conversation Transcript
-Date: 2019-12-02
+Date: 2019-01-03
```
Comment (bloom1) [inline content/blog/live-transcript-demo-astricon-19.md]: 2020!

Comment (sduthil) [reply content/blog/live-transcript-demo-astricon-19.md]: Good spot

**Pull request merged**